src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -65,8 +67,15 @@ const SetAclPermissionForm = () => {
         state,
         dispatch
       );
+      addNotification(
+        'info',
```

Comment:
Not too important at the moment with regards to coloring since that could be handled later, but I think we could use the color for 'success' for this snackbar.

Reply:
Adding onto this. I was also confused - but that's due to the code that was already there.   

So the fix for this is yes to make it `'success'` but also change the words to `Gave` and `Revoked` - I didn't initially realize this is after it is successfully completed.

Reply:
The more I think about it... this line:
`${permissions.read ? 'Give' : 'Revoke'} permission to ${permissionState.podUrlToSetPermissionsTo} for ${docType}.`

Should be:
`${permissions.read ? 'Gave' : 'Revoked'} permission ${permissions.read ? 'to' : 'from'} ${permissionState.podUrlToSetPermissionsTo} for ${docType}.`